### ISSUE TITLE
'Can't find anchor ... in ...' error when compiling docs with doc2dash

### ISSUE THREAD

**Issue by capac** [statsmodels/statsmodels.github.io#26]
When I build the _statsmodels_ docs for Dash, I usually get these types of errors, the `'Can't find anchor ... in ...'`. Below are some such examples. Even so, the docs are created, however what I've noticed is that some index entries are repeated, such as the entries for `sections` (see image below).

Is this a _statsmodels_ issue, or can it be confronted in _doc2dash_ ?

`Can't find anchor 'statsmodels.genmod.families.links.CLogLog' (EntryType.CLASS) in 'generated/statsmodels.genmod.families.links.CLogLog.html'.`

`Can't find anchor 'statsmodels.genmod.families.links.Log' (EntryType.SECTION) in 'generated/statsmodels.genmod.families.links.Log.html'.`

`Can't find anchor 'statsmodels.genmod.families.links.CLogLog.deriv' (EntryType.METHOD) in 'generated/statsmodels.genmod.families.links.CLogLog.deriv.html'.`

<img width="1035" alt="repeated-entries-in-sections" src="https://user-images.githubusercontent.com/541813/205020019-9b434e5e-311b-49e8-b09c-0e69fc1ce055.png">

**Comment by bashtage** [comment]
I don't know what doc2dash is, bit these are problems because there are both upper and lower case versions of the same links.  This means that case insensitive file systems can't render docs correctly.

**Comment by capac** [comment]
Indeed, my file system is case-insensitive. Is there a way in _statsmodels_ to retain just one link if multiple case sensitive links are available? As for _doc2dash_ (https://doc2dash.readthedocs.io/en/stable/), I don't know its parser well enough to figure this issue out. I'll ask on the corresponding GitHub issues page.

**Comment by bashtage** [comment]
Can you build on Linux+case sensitive or WSL?  This might be a solution.  For 0.14.0 I think the lowercase links are deprecated and should not appear in the docs.
